layouts/partials/openapi/render-request.html

```diff
@@ -52,11 +52,15 @@ <h3>Request body</h3>
             */}}
             {{ $mimes := slice }}
             {{ $descriptions := slice }}
+            {{ $added_in := slice }}
+            {{ $changed_in := slice }}
             {{ range $mime, $body := $request_body.content }}
                 {{ $mimes = $mimes | append $mime }}
                 {{ $descriptions = $descriptions | append $request_body.description }}
+                {{ $added_in = $added_in | append (index $request_body "x-addedInMatrixVersion") }}
+                {{ $changed_in = $changed_in | append (index $request_body "x-changedInMatrixVersion") }}
```

Comment:
Oh! So if requests and responses both have the same structure, then I think we don't need the slices to begin with. We can just pass e.g. `$request_body.content` into `{{ partial "openapi/render-content-type" ...` and do the looping on the inside. That's actually how I had implemented it originally but then realized that it didn't work for e.g.:

```
  requestBodies:
    bytes:
      content:
        application/octet-stream:
          example: <bytes>
      description: The content to be uploaded.
      x-addedInMatrixVersion: "1.7" <<< Added for testing
      required: true
```

Reply:
It should be added here but under `"application/octet-stream".schema`

Reply:
And yes, it _should_ work with passing `content`, but as I said, it would require to change this definition to have `description` under `"application/octet-stream".schema` too.

Reply:
Ok, that doesn't sound bad though. Let me give that a shot. Thanks for flagging this.

Reply:
Should be fixed with https://github.com/matrix-org/matrix-spec/pull/1876/commits/23393ad8491973e65ace3659164285b8f5c74517